jscripts/rating.js

```diff
@@ -88,7 +88,7 @@ var Rating = {
 		{
 			$.each(json.errors, function(i, error)
 			{
-				$.jGrowl(lang.ratings_update_error + ' ' + error);
+				$.jGrowl(lang.ratings_update_error + ' ' + error, {theme:'jgrowl_error'});
```

Comment:
Missed this: https://github.com/Sama34/mybb/blob/fix-1810/jscripts/rating.js#L100